hugo/content/info/corrections.md

```diff
@@ -69,3 +70,12 @@ Here are some good examples:
 Following these instructions will help us to process corrections more quickly.
 We will do our best to process revisions or errata within four weeks, but make no guarantees as to the processing time.
 Submissions not meeting these standards will be rejected, potentially without notice.
+
+Changes to the author list are potentially subject to further review.
```

Comment:
Maybe instead: "A revision that changes the author list needs permission (see below)."

"potentially subject to further review" does not sound like we would not have a specific process in place.